library/src/main/java/com/novoda/downloadmanager/lib/DownloadProvider.java

```diff
@@ -755,13 +752,6 @@ public int update(final Uri uri, final ContentValues values, final String where,
         int count;
         boolean startService = false;
 
-        if (values.containsKey(DownloadContract.Downloads.COLUMN_DELETED)) {
-            if (values.getAsInteger(DownloadContract.Downloads.COLUMN_DELETED) == 1) {
-                // some rows are to be 'deleted'. need to start DownloadService.
-                startService = true;
```

Comment:
No need to schedule a job anymore, the deletion happens immediately in the job itself